plugins/wpe-headless/.docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PHP_VERSION=7.4
-ARG WORDPRESS_VERSION=5.6
+ARG WORDPRESS_VERSION=5.6.2
 
 FROM wordpress:cli-php${PHP_VERSION} AS wp-cli
 FROM wordpress:${WORDPRESS_VERSION}-php${PHP_VERSION}-apache
```

Comment:
Any value in using `wordpress:latest` here so we always test against the latest WP without having to make manual version updates? Or is bumping manually desirable for stability?

It looks like the 'latest' tag is based on 7.4/apache at the moment. https://github.com/docker-library/wordpress/blob/b5d8e4f40bef55630fcf52a0f2830619f4df140e/latest/php7.4/apache/Dockerfile